jest-config/style-mock.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
what is this file about?

Reply:
required for successful tests using jest script (when some components import external style files with `.css`)